frontend/src/i18n/locales/de/frequent.json

```
@@ -0,0 +1,124 @@
+{
+  "Whoops! This page doesn't exist": "Huch! Diese Seite existiert nicht",
+  "Head back <1>home</1>.": "Zurück <1>nach Hause</1>.",
+  "Cancel": "Abbrechen",
+  "Something went wrong with cluster {{ cluster }}": "Mit dem Cluster ist etwas schief gelaufen {{ cluster }}",
+  "Choose another cluster": "Wählen Sie einen anderen Cluster",
+  "Skip to main content": "Zum Hauptinhalt springen",
+  "Notifications": "Nachrichten",
+  "Date": "Datum",
+  "Name": "Name",
+  "Server": "Server",
+  "Apply": "Anwenden",
+  "Delete": "Löschen",
+  "Custom value": "Benutzerdefinierter Wert",
+  "General": "Allgemein",
+  "Language": "Sprache",
+  "Theme": "Design",
+  "Settings": "Einstellungen",
+  "light theme": "helles Theme",
+  "dark theme": "dunkles Theme",
+  "Change theme": "Theme ändern",
+  "Log out": "Abmelden",
+  "(No token set up)": "(Kein Token eingerichtet)",
+  "Account of current user": "Konto des aktuellen Benutzers",
+  "Appbar Tools": "Appbar-Tools",
+  "show more": "mehr anzeigen",
+  "Version": "Version",
+  "Git Commit": "Git Commit",
+  "Try Again": "Erneut versuchen",
+  "Back": "Zurück",
+  "Next": "Weiter",
+  "Finish": "Beenden",
+  "Offline": "Offline",
+  "No": "Nein",
+  "Yes": "Ja",
+  "Toggle fullscreen": "Vollbild ein/aus",
+  "Close": "Schließen",
+  "Find": "finden",
+  "All namespaces": "Alle Namespaces",
+  "Type": "Typ",
+  "Reason": "Ereignis",
+  "From": "Von",
+  "Message": "Nachricht",
+  "Age": "Alter",
+  "Skip": "Überspringen",
+  "Read more": "Mehr lesen",
+  "Dismiss": "Ablehnen",
+  "Create / Apply": "Erstellen / Anwenden",
+  "Deleting item {{ itemName }}…": "Lösche Element {{ itemName }} …",
+  "Cancelled deletion of {{ itemName }}.": "Löschen von {{ itemName }} abgebrochen.",
+  "Deleted item {{ itemName }}.": "Element {{ itemName }} gelöscht.",
+  "Error deleting item {{ itemName }}.": "Fehler beim Löschen des Elements {{ itemName }}.",
+  "Edit": "Bearbeiten",
+  "Editor": "Editor",
+  "Documentation": "Dokumentation",
+  "Undo": "Rückgängig machen",
+  "Are you sure?": "Sind Sie sicher?",
+  "Undo Changes": "Änderungen rückgängig machen",
+  "Save & Apply": "Speichern & Anwenden",
+  "Loading resource data": "Lade Ressourcendaten",
+  "Mount Path": "Mount Path",
+  "from": "von",
+  "I/O": "I/O",
+  "Ready": "Bereit",
+  "Waiting": "Warten",
+  "Running": "Läuft",
+  "Error": "Error",
+  "Restart Count": "Anzahl der Neustarts",
+  "Restarting {{ itemName }}…": "Starte {{ itemName }}…",
```

```suggestion
  "Restarting {{ itemName }}…": "Starte {{ itemName }} neu…",
```
I realise this was my own mistake from the previous review 😅